src/settings/BatchGroupConfigurationSettings/BatchGroupConfigurationSettings.js

```diff
@@ -1,7 +1,86 @@
-import React from 'react';
+import React, {
+  useCallback,
+  useEffect,
+  useMemo,
+  useState,
+} from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
 
-const BatchGroupConfigurationSettings = () => {
-  return <div data-test-batch-group-configuration-settings />;
+import {
+  Pane,
+  Spinner,
+} from '@folio/stripes/components';
+
+import { batchGroupsResource } from '../../common/resources';
+import BatchGroupConfigurationSettingsForm from './BatchGroupCongiurationSettingsForm';
+
+const BatchGroupConfigurationSettings = ({ mutator }) => {
+  const [batchGroups, setBatchGroups] = useState();
+  const [batchGroupId, setBatchGroupId] = useState();
+  const paneTitle = <FormattedMessage id="ui-invoice.settings.batchGroupConfiguration.label" />;
+
+  useEffect(() => {
+    mutator.batchGroups.GET()
+      .then((batchGroupsResp) => {
+        const batchGroupIdResp = batchGroupsResp[0]?.id;
+
+        setBatchGroups(batchGroupsResp);
+        setBatchGroupId(batchGroupIdResp);
+      })
+      .catch(() => setBatchGroups([]));
+  },
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  []);
+
+  const batchGroupsOptions = useMemo(() => {
+    if (!batchGroups) return [];
+
+    return batchGroups.map(({ name, id }) => ({ label: name, value: id })) || [];
+  }, [batchGroups]);
+
+  const onSaveExportConfig = useCallback(
+    () => {
+
```

Comment:
is it supposed to have empty body?

Reply:
Just for now - `onSubmit` prop is required for form.